frontend/src/components/layout/UserDropDown.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Popper, Box, List, ListItem, ListItemText } from '@mui/material';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import {
+  UserInfo,
+  UserAvatar,
+  UserName,
+  UserPopper,
+  ArrowDown,
+  ArrowUp,
+  DividerHorizontal,
+  Link
+} from './styled'
+
+interface Props {
```

Comment:
Sort please the definition of your type. At first required, after that optional. It will be much more readable